README.md

```diff
@@ -43,19 +43,19 @@ npm i luxon @date-io/luxon
 npm i dayjs @date-io/dayjs
 ```
 
-Then teach pickers which library to use with `MuiPickerUtilsProvider`. This component takes a utils property, and makes it available down the React tree thanks to React context. It should preferably be used at the root of your component tree.
+Then teach pickers which library to use with `LocalizationProvider`. This component takes an adapter property, and makes it available down the React tree thanks to React context. It should preferably be used at the root of your component tree.
 
 ```jsx
-import MomentUtils from '@date-io/moment';
-import DateFnsUtils from '@date-io/date-fns';
-import LuxonUtils from '@date-io/luxon';
-import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import Luxon from '@date-io/luxon';
+import Moment from '@date-io/moment';
+import DateFns from '@date-io/date-fns';
+import { LocalizationProvider } from '@material-ui/pickers';
 
 function App() {
   return (
-    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+    <LocalizationProvider adapter={DateFns}>
```

Comment:
"adapter" might not make it clear that it's about handling dates, maybe?

```suggestion
    <LocalizationProvider date={dateFnsAdapter}>
```

Reply:
I think date is even more confusing, maybe like `dateLib={DateFns}` or  `dateLibAdapter={DateFns}`

Reply:
Yeah, you are right, `date` is probaby too short. A third possible option: `dateAdapter`.